app/models/garden.rb

```diff
@@ -4,6 +4,8 @@ class Garden
   belongs_to :user
   validates_presence_of :user
 
+  after_save :reindex_guides
```

Comment:
At the moment it's probably okay to re-index all the guides every time a garden changes, but does it make sense to do so once our guides DB starts getting larger?